test/parallel/test-stream-duplex-destroy.js

```diff
@@ -238,3 +238,20 @@ const assert = require('assert');
   });
   duplex.on('close', common.mustCall());
 }
+{
+  // Check abort signal
+  const controller = new AbortController();
+  const { signal } = controller;
+  const duplex = new Duplex({
+    write(chunk, enc, cb) { cb(); },
+    read() {},
+    signal,
+  });
+  let count = 0;
+  duplex.on('error', common.mustCall((e) => {
+    assert.strictEqual(count++, 0); // Ensure not called twice
```

Comment:
The mustCall already handles this doesn't it?

Reply:
@jasnell I want to make sure `error` is called, but not called twice - I'd need a `comon.mustCallOnce` or something.